src/codegen/ast_interpreter.cpp

```diff
@@ -630,6 +795,7 @@ Value ASTInterpreter::visit_invoke(AST_Invoke* node) {
         v = visit_stmt(node->stmt);
         next_block = node->normal_dest;
```

Comment:
oh yeah, there should be a finish call. I will add it.
I think even without it it should work fine but if the next basic block contains an instruction we don't support yet we will throw away more than necessary